openedx/core/djangoapps/user_api/accounts/tests/test_api.py

```diff
@@ -0,0 +1,144 @@
+# -*- coding: utf-8 -*-
+"""
+Unit tests for behavior that is specific to the api methods (vs. the view methods).
+Most of the functionality is covered in test_views.py.
+"""
+
+from mock import Mock, patch
+from django.test import TestCase
+import unittest
+from student.tests.factories import UserFactory
+from django.conf import settings
+from openedx.core.djangoapps.user_api.api.account import AccountUserNotFound, AccountUpdateError, AccountNotAuthorized
+from ..api import get_account_settings, update_account_settings
+
+
+def mock_render_to_string(template_name, context):
+    """Return a string that encodes template_name and context"""
+    return str((template_name, sorted(context.iteritems())))
+
+
+@unittest.skipUnless(settings.ROOT_URLCONF == 'lms.urls', 'Account APIs are only supported in LMS')
+class TestAccountApi(TestCase):
+    password = "test"
```

Comment:
Maybe add a docstring here that mirrors the module level one. I didn't look up to the top of the file to notice that you'd described why the API is only testing errors, and I think others might make that mistake.